### DISCUSSION TITLE
Apple silicon install issue

### DISCUSSION THREAD

**Discussion by 56phil** [LazyVim/LazyVim#2685] [General]
Hi,

This may be better as a bug report, but being new to LazyVim, I thought I'd start here.

I recently installed LazyVim on an older (Intel) Mac and loved it. Then, I went over to my newer (Apple Silicon) MacBook and it didn't go as well. When I started Neovim for the first time, things started out nicely but went south in a hurry. I don't remember any details. On the second, and subsequent activations of LazyVim, I get error messages. The first page is below. Has anyone else experienced anything similar? Does anyone have any ideas on how I can fix this? I someone would tell me where to find the log files, I would be happy to supply them. BTW, neo-tree (<leader>e) is also broke in this install. TIA

==================================================================
Failed to run 'config' for bufferline.nvim
vim/_init_packages. lua:0: module 'vim.version' not found:
'Ino field package preloadI 'vim.version']
cache_loader: module vim.version not found
cache_loader_lib: module vim. version not found
'Ino file ' vim version. lua'
^Ino file 'copt/homebrew/share/luajit-2.1/vim version.lua'
^Ino file 'cusr/local/share/lua/5.1/vim/version.lua'
^Ino file '/usr/ local/share/lua/5.1/vim/version/init.lua'
^Ino file 'copt/homebrew/share/lua/5.1/vim/version.lua'
^Ino file 'copt homebrew/share/lua/5.1/vim/version/init.lua'
"Ino file ''‹vim/version.so'
^Ino file 'cusr/local/lib/lua/5.1/vim/version.so'
"Ino file 'copt/homebrew/lib/lua/5.1/vim/version.so'
^Ino file 'cusr/local/lib/lua/5.1/loadall.so'
'Ino file ' vim.so'
^Ino file 'cusr/local lib/lua/5.1/vim.so'
^Ino file ^Ino file 'copt homebrew/lib/lua/5.1/vim.so'
'Ino file 'cusr/local/lib/lua/5.1/loadall.so'
# stacktrace:
- vim/_init_packages. lua:@ _in_ **__index**
- /bufferline.nvim/lua/bufferline/utils/init. lua:219 _in_ **is_current_stabl
- /bufferline.nvim/lua/bufferline. lua: 185 _in_ **setup**
- /LazyVim/lua/lazyvim/plugins/ui.lua:97 _in_ **config**
Press ENTER or type command to continue

**Comment by dpetka2001** [comment]
Relevant issues at `lazy.nvim` repo.
https://github.com/folke/lazy.nvim/issues/1214
https://github.com/folke/lazy.nvim/issues/648

It seems it doesn't have anything to do with LazyVim, rather it has to do with how git operates in newer Apple Macs. Don't have a Mac to test it.

**Comment by folke** [comment]
What @dpetka2001 says, or you are using a non-supported version of NEovim, since it can't find `vim.version`

**Comment by 56phil** [comment]
I think you two are right. Apple rearranged the file structure on the new Macs. Could it be that the clone command, or the repository, needs to be different to accommodate the difference? Anyway, thanks for the quick response.

NVIM V0.9.5
Build type: Release
LuaJIT 2.1.1703358377
system vimre file: "#VIM/sysinit.vim"
fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.9.5/share/nvim"'
Run :checkhealth for more info

**Reply by dpetka2001** [reply]
After a quick search based on the first issue that I linked in my previous comment, it seems that you can run your terminal under Rosetta as described [here](https://apple.stackexchange.com/questions/428768/on-apple-m1-with-rosetta-how-to-open-entire-terminal-iterm-in-x86-64-architec/428769#428769). Not sure if this will solve your problem, since I can't test it, but worth a try in my opinion.

**Comment by alewis001** [comment]
It's odd because I use neovim with lazyvim on an M1 Mac without any issues at all. I used homebrew to install neovim. I use git daily as well, again installed via homebrew. As far as I know, everything is using arm64 binaries instead of anything Rosetta based.

**Reply by 56phil** [reply]
Thanks. I suppose I could do a fresh install of MacOS and try again.

**Reply by 56phil** [reply]
I reinstalled Sonoma this afternoon and I'm happy to say that whatever the issue was, it's fixed now. alewis001 did me a solid. Thank you.

**Reply by alewis001** [reply]
Oh wow, I thought I was unfortunately giving you a "works for me", albeit with the best intentions. I'm glad it actually helped 😀
